src/core_plugins/kibana/public/home/components/tutorial_directory.js

```diff
@@ -37,29 +37,31 @@ import {
 
 import { getTutorials } from '../load_tutorials';
 
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
 const ALL_TAB_ID = 'all';
 const SAMPLE_DATA_TAB_ID = 'sampleData';
 
-export class TutorialDirectory extends React.Component {
+export class TutorialDirectoryUi extends React.Component {
```

Comment:
The export for this file is at the end now, so I think we don't want this one exported